packages/ai-chat/src/common/chat-service.ts

```diff
@@ -90,7 +116,7 @@ export type PinChatAgent = boolean;
 
 export const ChatService = Symbol('ChatService');
 export interface ChatService {
-    onActiveSessionChanged: Event<ActiveSessionChangedEvent>
+    onSessionEvent: Event<ActiveSessionChangedEvent | SessionCreatedEvent | SessionDeletedEvent>
```

Comment:
It probably doesn't hurt, or might be even useful to future listeners, but technically we don't use this anymore in this PR as far as I can see.